railties/lib/rails/generators/rails/benchmark/benchmark_generator.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "rails/generators/named_base"
+
+module Rails
+  module Generators
+    class BenchmarkGenerator < NamedBase
+      argument :reports, type: :array, default: []
+      class_option :method, type: :string, enum: %w(ips bm bmbm), default: "ips"
```

Comment:
How often do people actually use bm or bmbm? I thought everybody was just ips from now on?

Reply:
That is a good question. I personally have always used `ips` myself.

Is there value in keeping the different options present? The default is set to `ips` though.

Reply:
I like having the option of including `bm` and `bmbm` for cases where you only want to measure something once. That said I can't think of a common use case where that's really necessary, so maybe we can 🔥it and cut down the logic in the generator.

Reply:
I'm all for simplifying things. If we're good with this I'll pull out `bm` and `bmbm` from the generator.

Reply:
Sure, let's 🔥 it for now. We can always add it back later. That likely means `class_option :method` and all tests and docs associated with it can simply be removed.

Reply:
Agreed. Let's ship something more opinionated that steers people in a certain direction initially. And reap the implementation rewards of less scope for v1.